pkg/applications/providers/types.go

```diff
@@ -59,6 +59,12 @@ type TemplateProvider interface {
 
 	// Uninstall the application.
 	Uninstall(applicationInstallation *appskubermaticv1.ApplicationInstallation) (util.StatusUpdater, error)
+
+	// Check if a Release is stuck
+	IsReleaseStuck(applicationInstallation *appskubermaticv1.ApplicationInstallation) (bool, error)
```

Comment:
Nit: Should this be called `IsReleaseStuck` or maybe just `IsStuck`? Given that we potential can add providers that aren't Helm, the "Release" part of this doesn't make much sense. It's a very Helm-specific term.

Reply:
good point, have renamed it